schemas/README.md

```diff
@@ -1,13 +1,122 @@
 
-Files and directories:
-- [schema/](./): reference schemas for all model parameters (one file per parameter) and input data required by the derivation tools
-- [schema/jsonschema.yml](./jsonschema.yml): json schema for validation of all schema files
+## Typical Example
 
-## Technical notes
+The following example is typical for a single parameter description.
 
-For easy generation of json schema files from yaml example, see https://codebeautify.org/yaml-to-json-schema-generator
+```yaml
+%YAML 1.2
+---
+title: Schema for num_gains model parameter
+version: 0.1.0
+base_schema: simpipe-schema
+base_schema_url: https://github.com/gammasim/workflows/schema/jsonschema.yml
+base_schema_version: 0.1.0
+name: num_gains
+description: Number of different gains the input signal gets digitized.
+data:
+  - type: int
+    units: dimensionless
+    default: 1
+    allowed_range:
+      min: 1
+      max: 2
+instrument:
+  class: Camera
+  type:
+    - LSTCam
+    - FlashCam
+    - NectarCam
+    - SSTCam
+    - SCTCam
+activity:
+  setting:
+    - SetParameterFromExternal
+  validation:
+    - ValidateParameterByExpert
+source:
+  - Initial instrument setup
+simulation_software:
+  - name: sim_telarray
+```
+
+## Valid Keys
+
+Valid keys are described in detail in [jsonschema.yml](./jsonschema.yml). The following list gives a short overview.
+
+### Header section
+
+- `title`: Title of the schema file.
+- `version`: Version of this schema file.
+- `base_schema`: Name of the base schema.
+- `base_schema_url`: URL of the base schema.
+- `base_schema_version`: Version of the base schema.
+
+### Parameter description
+
+- `name`: Name of the parameter.
+- `description`: Description of the parameter.
+- `short_description`: Short description of the parameter.
+
+### Parameter data
+
+The *data* field is used to describe the actual type, format, and allowed values of the parameter.
+
+- `type`: Data type of the parameter.
+- `units`: Units of the parameter (compatible with astropy units). Note that units are explicitly listed in the jsonschema.yml file.
+- `default`: Default value of the parameter.
+- `allowed_range`: Allowed range of the parameter.
+- `allowed_values`: Allowed values of the parameter. Use if values are not continuous and restricted to a small list of allowed values.
+
+Tables of any dimensions can be described using the *datatable* field. A simple example is e.g.
+
+```yaml
+data:
+  - type: datatable
+    table_columns:
+      - name: wavelength
+        description: Wavelength.
+        required: true
+        units: nm
+        type: double
+        required_range:
+          min: 300.0
+          max: 700.0
+        input_processing:
+          - remove_duplicates
+          - sort
+      - name: photo_detection_efficiency
+        description: Average quantum or photon detection efficiency.
+        required: true
+        units: dimensionless
```

Comment:
Related to the previous comment: I know that you prefer it to be more human readable, but note that if this is supposed to be astropy compatible, I think this should be`!astropy.units.Unit {unit: ''}`. (the same applies for all the units)

Reply:
Ah, now I see that the allowed units are also listed in `schemas/jsonschema.yml`. Please mention that somewhere in the text as well.

Reply:
But astropy understands "km" or "dimensionless_unscaled" (see discussion #9). I really don't see a reason to make it look more complicated than it has to.

The comment that units a required to be listed (this is not a choice; json-schema requires it) is mentioned further down.